ts/features/zendesk/saga/index.ts

```diff
@@ -117,7 +111,7 @@ function* getZendeskTokenSaga(
     const response = (yield* call(
       withRefreshApiCall,
       getSession({ fields }),
-      getZendeskToken.failure("401") // if the error is 401 the error screen is not show thanks this parameter
+      getZendeskToken.request()
```

Comment:
It's safe to remove `getZendeskToken.failure("401")` from here?

Reply:
Yes! As you can see in the screen recordings of the as-is in case there is a 401, the flow picks up where it left off